README.md

```diff
@@ -55,9 +51,9 @@ LDK/`rust-lightning` is a generic library which allows you to build a Lightning
 node without needing to worry about getting all of the Lightning state machine,
 routing, and on-chain punishment code (and other chain interactions) exactly
 correct. Note that `rust-lightning` isn't, in itself, a node. There are various
-working/in progress demos which could be used as a node today, but if you "just"
-want a generic Lightning node, you're almost certainly better off with [Core
-Lightning](https://github.com/ElementsProject/lightning) or
+nodes built using LDK such as [Sensei](https://l2.technology/sensei), but if you
+"just" want a generic Lightning node, you're almost certainly better off with
+[Core Lightning](https://github.com/ElementsProject/lightning) or
 [LND](https://github.com/lightningnetwork/lnd). If, on the other hand, you want
 to integrate Lightning with custom features such as your own chain sync, your
```

Comment:
Better ordering in-line with user needs could be :
 "On the other hand,  if you want to integrate Lightning with swappable components such as your own storage layer, custom key management, your own routing or chain-sync etc., LDK is likely your only option."
 
 or something around those lines.